src/generalChecks/index.js

```diff
@@ -72,5 +73,36 @@ module.exports = testPath => {
       return;
     }
     console.error(green, 'Existe un archivo .babelrc');
+
+    read(`${testPath}/.babelrc`, 'utf8', (err, data) => {
+      const moduleResolver = JSON.parse(data).plugins.filter(
+        plugin => Array.isArray(plugin) && plugin[0] === 'module-resolver'
+      );
+      if (!moduleResolver.length) {
+        console.log(red, 'El archivo .babelrc no contiene el plugin "module-resolver"');
+        return;
+      }
+      const aliases = moduleResolver[0][1].alias;
+      const isBaseAlias = alias =>
+        Object.keys(aliases).includes(alias) || console.log(red, `Falta absolute import para: ${alias}`);
+      const validPath = alias =>
+        aliasPathRegex(alias).test(aliases[alias]) ||
+        console.log(red, `El import absoluto de "${alias}" no está configurado correctamente`);
+      if (
+        BASE_ALIASES.map(alias => isBaseAlias(alias) && validPath(alias)).reduce(
+          (accumulator, current) => accumulator && current
```

Comment:
I think the `map` is unnecesary 🤔

Reply:
Do you mean that it will be enough using only reduce?